submissions/sprint_b/app.py

```diff
@@ -0,0 +1,36 @@
+from sprint_b import app
+import json
+from helpers import EpithetGenerator, Vocabulary
+
+path = ('/Users/ethansternke/Documents/Kenzie/back-end'
+        '/backend-epithet-generator/resources/data.json')
+
+
+@app.route('/')
+def generate_epithet():
+    e = EpithetGenerator()
+    epithet = e.generate_epithet(path)
+    response = {"epithet": epithet}
+    return json.dumps(response)
+
+
+@app.route('/vocabulary')
+def vocabulary():
+    v = Vocabulary()
+    vocabulary = v.from_file(path)
+    response = {"vocabulary": vocabulary}
+    return json.dumps(response)
+
+
+@app.route('/epithets/<quantity>')
+def generate_epithets(quantity):
+    e = EpithetGenerator()
+    epithets = []
+    for i in range(int(quantity)):
```

Comment:
Your generate method should accept the quantity of epithets to generate 
and return a list of epithets. All of the logic for looping some quantity of 
times should be part of your generate method and not defined in the view. This 
DRYs up the view so you don't have to repeat iteration logic in each view. 

Define an EpithetGenerator class with methods to:
select one random word from each column of the list.
generate a quantity of epithets from a vocabulary file loaded from a path.